libs/dataset-explorer/src/lib/DistributionBalanceMeasuresChart.tsx

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { Link, Stack, Text } from "@fluentui/react";
+import {
+  BasicHighChart,
+  IDistributionBalanceMeasures,
+  LabelWithCallout
+} from "@responsible-ai/core-ui";
+import { localization } from "@responsible-ai/localization";
+import React from "react";
+
+import { dataBalanceTabStyles } from "./DataBalanceTab.styles";
+import { getDistributionBalanceMeasuresChart } from "./getDistributionBalanceMeasuresChart";
+
+export interface IDistributionBalanceMeasureProps {
+  distributionBalanceMeasures: IDistributionBalanceMeasures;
+}
+
+export class DistributionBalanceMeasuresChart extends React.PureComponent<IDistributionBalanceMeasureProps> {
+  public render(): React.ReactNode {
+    const distributionBalanceMeasures = this.props.distributionBalanceMeasures;
+    if (!distributionBalanceMeasures) {
+      return;
+    }
```

Comment:
Should we point out to users that they'd need to upload these / pass them in to view this type of chart? Particularly, already existing uploaded dashboard data won't include this so it's a case users will actually hit.